src/app/data-explorer/selection-compact/selection-compact.component.html

```diff
@@ -42,6 +39,7 @@ <h3>Current selection</h3>
       <mdm-data-element-multi-select
         [sourceTargetIntersections]="sourceTargetIntersections"
         [dataElements]="(selectedElements$ | async) ?? []"
+        (selectionAddedToSpec)="clearSelection()"
```

Comment:
`mdm-data-element-multi-select` is used in other parts of the code and the `(selectionAddedToSpec)` has not been added to these definitions which means the selection IS NOT cleared when using those parts. E.g, this one does not clear the spec list after being clicked.

![image](https://github.com/user-attachments/assets/024a6283-ecb6-482d-a2dc-9000e36f126f)

**Extra Task:** The dialog in the screenshot above also does not disappear after the spec has been updated, even if the user clicks on "Open Data Specification" after the fields have been added. Could we please fix this whilst we are working in this area.

Reply:
Added the clearSelection trigger here, but not on bookmarks page, since that selection is not persistent anyway and user may find it helpful to add a small set of fields to multiple specs (not now possible from the main browse)

Extra Task seems to have resolved itself by virtual of clearing the selection